telegraf.js

```diff
@@ -5,6 +5,8 @@ const Composer = require('./composer')
 const Markup = require('./markup')
 const session = require('./session')
 const Router = require('./router')
+const Stage = require('./stage')
+const BaseScene = require('./scenes/base')
```

Comment:
I think we don't need to reexport Stage stuff here because node will parse these code even if we don't need it

Reply:
But we can't add separate typedef for stages/scenes because of the way typescript works